packages/react-next/src/components/TextField/MaskedTextField/MaskedTextField.tsx

```diff
@@ -15,385 +14,374 @@ import {
   insertString,
   parseMask,
 } from './inputMask';
+import { useConst } from '@uifabric/react-hooks';
+
+interface IMaskedTextFieldState {
+  maskCharData: IMaskValue[];
+  isFocused: boolean;
+  moveCursorOnMouseUp: boolean;
+  changeSelectionData: {
+    changeType: 'default' | 'backspace' | 'delete' | 'textPasted';
+    selectionStart: number;
+    selectionEnd: number;
+  } | null;
+}
 
-/**
- * State for the MaskedTextField component.
- */
-export interface IMaskedTextFieldState {
+interface IMaskedTextField extends ITextField {
   /**
-   * The mask string formatted with the input value.
-   * This is what is displayed inside the TextField
-   * @example
-   *  `Phone Number: 12_ - 4___`
+   * The value of all filled format characters, or undefined if not all format characters are filled.
    */
-  displayValue: string;
-  /** The index into the rendered value of the first unfilled format character */
-  maskCursorPosition?: number;
+  value: string | undefined;
+  setValue: (newValue: string) => void;
```

Comment:
I think we actually should *not* expose this on the public interface (and only add it back if someone complains)--exposing a setter separate from props is generally not a good pattern for React components.